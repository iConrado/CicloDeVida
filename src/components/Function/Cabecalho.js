import React from 'react';
import { StyleSheet, View, Text } from 'react-native';

const styles = StyleSheet.create({
  cab: {
    backgroundColor: '#568CD9', 
    flex: 1, 
    width: '100%',
    justifyContent: 'center',
    alignItems: 'center',
  },
  tex: {
    fontSize: 22,
    fontWeight: 'bold',
    color: '#454545',
  },
});

const Cabecalho = (texto) => 
  (<View style={styles.cab}>
    <Text style={styles.tex}>{texto}</Text>
  </View>
); 

export default Cabecalho;