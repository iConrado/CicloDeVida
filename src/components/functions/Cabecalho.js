import React from 'react';
import { StyleSheet, View, Text } from 'react-native';

// Estilo a ser usado no componente principal do cabeçalho. INTERNO
const styles = StyleSheet.create({
  cab: {
    backgroundColor: '#0A2955', 
    flex: 1, 
    width: '100%',
    justifyContent: 'center',
    alignItems: 'center',
  },
  tex: {
    fontSize: 22,
    fontWeight: 'bold',
    color: '#FFF',
  },
});

// Cabeçalho padrão. EXPORT PADRÃO
const Cabecalho = (texto) => 
  (<View style={styles.cab}>
    <Text style={styles.tex}>{texto} 0.1.17</Text>
  </View>
); 

export default Cabecalho;
