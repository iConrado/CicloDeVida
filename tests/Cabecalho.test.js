import 'react-native';
import renderer from 'react-test-renderer';

import Cabecalho from '../src/components/functions/Cabecalho';

it('renders correctly', () => {
  const rendered = renderer.create(Cabecalho('teste')).toJSON();
  expect(rendered).toMatchSnapshot();
});
