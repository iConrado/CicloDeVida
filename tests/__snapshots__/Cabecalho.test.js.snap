// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#0A2955",
      "flex": 1,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#FFF",
        "fontSize": 22,
        "fontWeight": "bold",
      }
    }
  >
    teste
     0.1.21
  </Text>
</View>
`;
